absl/copts/AbseilConfigureCopts.cmake

```diff
@@ -11,7 +11,7 @@ if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64" OR "${CMAKE_SYSTEM_PROCESSOR}"
   else()
     set(ABSL_RANDOM_RANDEN_COPTS "${ABSL_RANDOM_HWAES_X64_FLAGS}")
   endif()
-elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "arm")
+elseif("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "arm.*|aarch64")
```

Comment:
Is there any specific reason to use regex matches rather than just `STREQUAL arm OR STREQUAL(aarch64)`?

Reply:
Good question, yes there is. 

On unix systems `CMAKE_SYSTEM_PROCESSOR` is set to the output of `uname -p` which, after some googling through some common ARM based boards (rpi3, odroid) I see the arm instruction set version appended at the end, e.g. `armv7l`, `armv6l`. I'm also guessing the original author had a system where `CMAKE_SYSTEM_PROCESSOR` was set to `arm` so the regex `arm.*` should match all these possibilities whereas `STREQAL arm` wouldn't due to it not being an exact match.